cli/lib/png2src.js

```diff
@@ -37,11 +38,17 @@ const %name%Height = %height%
 const %name%Flags = %flags% // %flagsHumanReadable%
 var %name% = [%length%]byte { %bytes% }`,
 
+    odin: 
+`%odinName%_width : u32 : %width%
+%odinName%_height : u32 : %height%
+%odinName%_flags : w4.Blit_Flags : %odinFlags% // %flagsHumanReadable%
```

Comment:
Any reason these are not not in upper case?

And the `u32` can be dropped for the width and height to keep it untyped.

Reply:
By reading the examples of Odin, I was pretty sure it uses lower `snake_case`. That is `name_of_sprite` instead of Rusts `NAME_OF_SPRITE`. Was I wrong here?

About the type: Yes, I tested that. We can, in fact, drop the length too. Thing is: The same is true for Go. And it's still in the template. If we change it for Odin, I think we should also change it for other languages that support it.